Missing navigation added

MilicaBokonjic opened the pull request.

thomashauser reviewed (changes requested):
- Navigation not working
- Should be part in General Design area?

MilicaBokonjic commented:
> * Navigation not working
> * Should be part in General Design area?

Asked Kerstin, she agrees. Moved it. And fixed copy paste error in navigation
